bin/dev/test_subaseg.py

```diff
@@ -1,4 +1,34 @@
+# -*- coding: utf-8 -*-
+# Tools for building full brain network models from standard structural MR scans.
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2017, Baycrest Centre for Geriatric Care ("Baycrest") and others
```

Comment:
See prev comment about copyright notice

Reply:
ok

Reply:
Can you please look again i remove all the license